apm-sniffer/apm-agent-core/pom.xml

```diff
@@ -104,6 +104,12 @@
             <version>${ststem-rules.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-generator-annprocess</artifactId>
+            <version>${jmh.version}</version>
```

Comment:
This is not required, right? As you have used the dependency management.

Reply:
Yes. Removed now